Documentation/CodingStyle.md

```diff
@@ -1,24 +1,23 @@
 # Coding Style
 
 Please consider using an [EditorConfig](http://editorconfig.org/)-capable
-text-editor. Plug-ins exist for most popular text-editors. This ensures
+text editor. Plug-ins exist for most popular text editors. This ensures
 that our [configuration file](../.editorconfig) gets respected, which
 reduces needless commit-noise.
 
-Generally, try to follow the local code-style of the source-file you're
-editing. It's much more disturbing to swtich between multiple code-styles
-within a source-file, than between source-files.
+Generally, try to follow the local code style of the source file you're
+editing. It's much more disturbing to swtich between multiple code styles
```

Comment:
Missed a `swtich`!

Reply:
Whoops, good catch! Seems this is a common typo, it's two other places in the fuselibs source!

> $ git grep swtich
> Documentation/CodingStyle.md:editing. It's much more disturbing to swtich between multiple code styles
> Source/Fuse.Navigation/WhileNavigationTriggers.uno:                     As the user swipes from B to C the progress of B will reduce from 1 towards 0 and the progress of C increases from 0 towards 1. The `Threadhold="0.5"` here causes the trigger swtich at the mid-way point of the transition. In this setup this means that one page is green and the rest are red -- the one closest to active is green.
> Source/Fuse.Triggers/StateGroup.uno:                    //this is needed in case a state swtich results in a state switch, see `ChainedSwitch` test case